### ISSUE TITLE
[Bug]: Can't get device metadata for long NHP neuropixel

### ISSUE THREAD

**Issue by aidapiccato** [catalystneuro/neuroconv#685]
### What happened?

`get_device_metadata` in `neuroconv/datainterfaces/ecephys/spikeglx/spikeglx_utils.py` failed to generate a probe type description for a spikeglx meta file with "imDatPrb_type" of 1030 (which is the probe type for the 45 mm NHP neuropixels). 

I think this would be resolved by adding an entry to the dictionary on line 90 with key "1030" and an appropriate descriptor (e.g. "NP1.0(NHP 45mm)")

### Steps to Reproduce

```python
processed_converter = PiccatoNWBConverter(
        source_data=processed_source_data,
        sync_dir=str(data_dir / "sync_signals"))


metadata = processed_converter.get_metadata()
```

### Traceback

```python
Traceback (most recent call last):
  File "/net/vast-storage/scratch/vast/jazayeri/apiccato/jazayeri-lab-to-nwb/src/jazayeri_lab_to_nwb/piccato/piccato_convert_session.py", line 272, in <module>
    session_to_nwb(
  File "/net/vast-storage/scratch/vast/jazayeri/apiccato/jazayeri-lab-to-nwb/src/jazayeri_lab_to_nwb/piccato/piccato_convert_session.py", line 199, in session_to_nwb
    metadata = processed_converter.get_metadata()
  File "/om2/user/apiccato/anaconda/envs/jazayeri_lab_to_nwb_env/lib/python3.10/site-packages/neuroconv/nwbconverter.py", line 86, in get_metadata
    interface_metadata = interface.get_metadata()
  File "/om2/user/apiccato/anaconda/envs/jazayeri_lab_to_nwb_env/lib/python3.10/site-packages/neuroconv/datainterfaces/ecephys/spikeglx/spikeglxdatainterface.py", line 100, in get_metadata
    device = get_device_metadata(self.meta)
  File "/om2/user/apiccato/anaconda/envs/jazayeri_lab_to_nwb_env/lib/python3.10/site-packages/neuroconv/datainterfaces/ecephys/spikeglx/spikeglx_utils.py", line 82, in get_device_metadata
    probe_type_description = probe_type_to_probe_description[probe_type]
KeyError: '1030'
```

### Operating System

Linux

### Python Executable

Python

### Python Version

3.9

### Package Versions

[environment_for_issue.txt](https://github.com/catalystneuro/neuroconv/files/13667087/environment_for_issue.txt)

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/catalystneuro/neuroconv/blob/main/.github/CODE_OF_CONDUCT.rst)
- [X] Have you ensured this bug was not already [reported](https://github.com/catalystneuro/neuroconv/issues)?

**Comment by CodyCBakerPhD** [comment]
@aidapiccato Sorry for the delayed response - working on this now

I don't see any v1 long-shank NHP probe examples on GIN (maybe @samuelgarcia or @alejoe91 or @h-mayorquin can correct me): https://gin.g-node.org/NeuralEnsemble/ephy_testing_data/src/master/spikeglx

So if not I'd really love to see some examples; would you be able to record and share a small snippet? @nwatters01 should be able to help with upload if it's your first time doing it through the MIT side

**Comment by samuelgarcia** [comment]
Hi Codey.
We have more meta file for spikeglx with more probe here https://github.com/SpikeInterface/probeinterface/tree/main/tests/data/spikeglx
if you want to make tests.
We do not have the relative binary file.

Maybe the one you look for is alreay in this folder.

**Comment by aidapiccato** [comment]
[This](https://github.com/SpikeInterface/probeinterface/blob/main/tests/data/spikeglx/non_human_primate_long_staggered.imec0.ap.meta) is for the correct probe but a different version of spikeglx. If a .meta file is sufficient, I can share on this thread directly. Otherwise, I'll record and upload a small snippet.

**Comment by samuelgarcia** [comment]
Hi Aida. Thank you.
If it is in probeinterface then this probe type is handle by probeinterface normaly.
For neuroconv I am not sure.

**Comment by CodyCBakerPhD** [comment]
fixed by #701
